src/aat/resources/features/F-044/S-579_Get_Case_For_Case_Worker_Data_Base.td.json

```diff
@@ -0,0 +1,80 @@
+{
+  "_guid_": "S-579_Get_Case_For_Case_Worker_Data_Base",
+  "productName": "CCD Data Store",
+  "operationName": "Get Case for Case Worker",
+  "method": "GET",
+  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}",
+  "user": {
+    "_extends_": "Common_User_For_Request"
+  },
+  "request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers"
+    },
+    "pathVariables": {
+      "cid": "${[scenarioContext][parentContext][childContexts][Befta_Jurisdiction2_Default_Full_Case_Creation_Data][testData][actualResponse][body][id]}",
+      "uid": "[[DYNAMIC]]",
+      "jid": "BEFTA_JURISDICTION_2",
+      "ctid": "BEFTA_CASETYPE_2_1"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Type": "[[ANY]]",
+      "Content-Length": "[[ANY]]",
+      "Content-Encoding": "gzip"
+    },
+    "body" : {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": "[[ANY]]",
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
+        "MoneyGBPField": "111100",
```

Comment:
Can use inheritance.

Reply:
we had tried but not possible at this moment

Reply:
No, we can't use Standard_Full_Case_Data as the data is different